pkgs/by-name/li/libloot/package.nix

```
@@ -0,0 +1,166 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  rustPlatform,
+  rust-cbindgen,
+
+  cmake,
+  pkg-config,
+
+  doxygen,
+  python3,
+
+  boost,
+  fmt_11,
+  gtest,
+  icu,
+  spdlog,
+  tbb_2021_11,
+  yaml-cpp,
+}:
+
+let
+  testing-plugins = fetchFromGitHub {
+    owner = "Ortham";
+    repo = "testing-plugins";
+    rev = "refs/tags/1.6.2";
+    hash = "sha256-3Aa98EwqpuGA3YlsRF8luWzXVEFO/rs6JXisXdLyIK4=";
+  };
+
+  buildRustFFIPackage =
+    args:
+    rustPlatform.buildRustPackage (
+      args
+      // {
+        postConfigure = ''
+          cp -r --no-preserve=all ${testing-plugins} testing-plugins
+        '';
+
+        nativeBuildInputs = [ rust-cbindgen ];
+
+        buildAndTestSubdir = "ffi";
+
+        postBuild = ''
+          cbindgen ffi/ -l $lang -o $out/include/$header
+        '';
+      }
+    );
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libloot";
+  version = "0.24.1";
+
+  src = fetchFromGitHub {
+    owner = "loot";
+    repo = "libloot";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Y0YYPlfCkXW+ONmXZ/C/jV/VEkagXzt3qHhN5zjxBUw=";
+  };
+
+  patches = [ ./deps.patch ];
```

```suggestion
  patches = [
    # Comment explaining why we need the patch
    ./deps.patch
  ];
```

Could you add a comment explaining the reason for the patch?